lib/semantics/check-omp-structure.cc

```diff
@@ -404,6 +404,15 @@ void OmpStructureChecker::Enter(const parser::OpenMPBlockConstruct &x) {
         OmpClause::DEVICE, OmpClause::DEFAULTMAP, OmpClause::NOWAIT};
     SetContextAllowedOnce(allowedOnce);
   } break;
+  case parser::OmpBlockDirective::Directive::Teams: {
```

Comment:
For other directives, the clauses are listed in comments before the case statement. Could you do the same here?

Reply:
Good point. I even made a mental note to do this when I went off to write the tests to check it, then never came back and did it..